app/models/initiative.rb

```diff
@@ -49,15 +49,15 @@ class Initiative < ApplicationRecord
   validate :must_have_manager
 
   def must_have_manager
-    if not (
-        source.is_a?(Manager) or
-        helpers.any? { |h| h.is_a?(Manager) }
-      )
-        errors.add :wyeworker_initiative_belongings, "An initiative must have a manager involved, as a source or as a helper"
+    unless source.is_a?(Manager) ||
```

Comment:
It's not recommended to use unless with multiple conditions since it can be hard to follow. Anyways an option to keep using unless could be using an auxiliar method.
It could be something like
`unless manager_is_involved`

```
def manager_is_involved
  source.is_a?(Manager) || helpers.any? { |h| h.is_a?(Manager) }
end
```

Reply:
Got it! Rubocop by default preferred `unless`, but I configured it!